### PR TITLE
plat-sam: matrix: fix warnings found during checkpatch

### PR DESCRIPTION
**Pull request by TonyHan11**

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
This pull request is a bit large to be reviewed in one go. Please create smaller pull requests so we can take this in smaller increments. About 5 commits and fewer than 1000 lines per pull request is a good target. That should help get this through.

Feel free to keep this pull request open to serve as a reference.

**Comment by tprrt** [comment]
Hello @jenswi-linaro,

What do you recommend for cutting?

Kind regards,
Thomas

**Comment by jenswi-linaro** [comment]
> What do you recommend for splitting?

Exactly what is not so important. It's fine to start with the first ones, but it's nice if they have some common topic. The point is to keep a small scope for each PR since it's easier to get a smaller PR merged without long delays.

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

File: core/arch/arm/plat-sam/matrix.c [hunk 1]
```diff
@@ -510,7 +510,7 @@ static unsigned int matrix_read(int base, unsigned int offset)
 void matrix_write_protect_enable(unsigned int matrix_base)
 {
 	matrix_write(matrix_base, MATRIX_WPMR,
-		    (MATRIX_WPMR_WPKEY_PASSWD | MATRIX_WPMR_WPEN_ENABLE));
+		     (MATRIX_WPMR_WPKEY_PASSWD | MATRIX_WPMR_WPEN_ENABLE));
```
Comment (etienne-lms) [inline core/arch/arm/plat-sam/matrix.c:513]: my 2 cents: parentheses not needed.

Comment (TonyHan11) [reply core/arch/arm/plat-sam/matrix.c]: Hi @etienne-lms,
Updated by removing the parentheses, thank you.

**Comment by etienne-lms** [comment]
rebase on master tip to get IBART test failure fixed.

**Comment by TonyHan11** [comment]
Rebased the commit on the master.

**Review by etienne-lms — COMMENTED** [review]
`Acked-by: Etienne Carriere <etienne.carriere@foss.st.com>`
Please apply @jenswi-linaro review tag also.

**Comment by TonyHan11** [comment]
Added review & ack tags.

**Pull request merged**